filament/backend/include/backend/Platform.h

```diff
@@ -61,6 +61,24 @@ class UTILS_PUBLIC Platform {
         MULTIVIEW,
     };
 
+    /**
+     * The action the Metal backend should take if a Drawable cannot be acquired.
+     *
+     * Each frame rendered requires a CAMetalDrawable texture, which is presented on-screen at the
+     * completion of each frame. These are limited and provided round-robin style by the system.
+     */
+    enum class MetalDrawableFailureBehavior : uint8_t {
```

Comment:
this should be in a public `MetalPlatform` header that the concret  metal implementation derives from. just like we did for `OpenGLPlatform`